src/Stryker.CLI/Stryker.CLI/runtimeconfig.template.json

```diff
@@ -1,3 +1,3 @@
 {
-  "rollForwardOnNoCandidateFx": 2
-}
\ No newline at end of file
+ // "rollForwardOnNoCandidateFx": 2
```

Comment:
Removing this causes a 1-1 dependency on dotnet runtime version and a error on missing runtime even if a newer and probably compatible runtime is available.

Reply:
oh shit. Thanks for pointing that out. I had to comment it  when I tried VsStudio 17.7 preview to look into the issue #2585, as I had errors due to conflicting settings regarding framework preferences. 
BTW I am keeping the preview version and removed the regular one. It looks to me like it obsoletes several things in the build architecture. For now, I can no longer build NFluent solution via MsBuild (but not problem from VS itself)

Reply:
new update: it turns out the issue was simpler: Stryker does not detect preRelease version, and it is the only version I kept. So it fell back to MsBuild 4, which was too old to support the project.
So I added detection of prerelease version (if normal fails) and ensure that the msbuild fallback message is displayed (information instead of debug)